src/tasks/network/killProcessUsingPort.js

```diff
@@ -0,0 +1,58 @@
+const { ask } = require('KegQuestions')
+const { getProcessesUsingPort, printProcesses, kill } = require('KegUtils')
+const { generalError } = require('KegUtils/error')
+
+/**
+ * Asks user to confirm that they want the processes killed
+ * @param {string} port
+ * @param {Array<Object>} processes - processes to confirm to kill
+ * @returns {Promise<bool>} - user response (true if you should kill the processes)
+ */
+const confirmKill = async (port, processes=[]) => {
+  printProcesses(`Processes using port ${port}:`, processes)
+  return await ask.confirm(`Kill ${processes.length} processes? [y/n]`)
```

Comment:
Not a big deal here, but you don't need to do `return await ...`. Just return, and what ever calls this method can handle it how it wants.
Doing like this means, it won't return until the await process has finished.
By just returning, then the caller can decide to await or not

Reply:
yeah you are totally right

updated